examples/favorites_advanced/getx_favorites_advanced/lib/feature_home/controllers/navbar_controller.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:get/get.dart';
+import 'package:favorites_advanced_base/models.dart';
+import 'package:favorites_advanced_base/extensions.dart';
+
+class NavbarController extends GetxController {
```

Comment:
@RomanovaPrime, could you please add unit tests for the business layer of the app ?

Reply:
Added unit tests for both controllers.